test/e2e/providers/vsphere/vsphere.go

```diff
@@ -110,17 +110,20 @@ func (p *Provider) getWorkspaceFromExistingMachineSet() (*mapi.Workspace, error)
 
 // getNetwork returns the network that needs to be used in the MachineSet
 func getNetwork() string {
-	// Default network for dev environment
-	networkSegment := "devqe-segment-222"
+	// $LEASED_RESOURCE holds the network the CI cluster is in and allows configuration of the network segment
+	// to use for testing with development clusters.
```

Comment:
I don't follow how `LEASED_RESOURCE` works in conjunction with CI and development clusters. Is this an environment variable that gets set in a CI env i.e., with e2e tests or are you introducing this for dev testing? Or both?

If both, do we need to complicate our workflow even further? Can't we just agree to use `devqe-segment-222`? If yes, then this commit can be dropped and the only commit needed will be to update the default network.

Reply:
Correct, `devqe-segment-222` could be agreed as the default, and remove this commit. 

That is the main goal of this PR, propose `devqe-segment-222` as the new default network segment. I wanted to explore with the team the need to configure the network segment externally. Will drop the commit.